docs/appendix/yaml.rst

```diff
@@ -428,7 +478,26 @@ observation label is first and the model label is second
    * **ts_avg_window:** This is for timeseries plots only. This is the averaging 
      window applied to the data. No averaging done if not provided in the yaml file (i.e., ts_avg_window is optional). Averaging is done if a pandas 
      resample rule (e.g., 'H' is hourly, 'D' is daily) is specified.
-   
+   * **altitude_yax2:** This is for "timeseries" plots for aircraft evaluation only. 
+     This allows users to add a secondary right y-axis to the timeseries plot. Additional YAML options 
+     are described below:
+     
+      * **altitude_variable:** Name of altitude variable in observational 
+        dataset (e.g., altitude)
+      * **altitude_ticks:** Option to set tick increments (e.g., 1000)
+      * **ylabel2:** Option to set secondary right y-axis label (e.g., Altitude (m))
+      * **plot_kwargs_y2:** Optional kwargs for secondary right y-axis (e.g., color: g)
+      * **filter_dict:** Optionally provide the following options to specify the min and max 
+        values of altitude for the secondary right y-axis
```

Comment:
With the nesting, the keys under it could be simplified, e.g.
- altitude_variable -> variable
- altitude_ticks -> ticks
- ylabel2 -> ylabel
- plot_kwargs_y2 -> plot_kwargs

Just a thought.

Reply:
That's a good point. I have an issue to clean up some of this in Version 2 and I will add this there, but not enough time to fix it this week.